### DISCUSSION TITLE
ch6-cloud/ch6-07-crawler

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#633] [General]
# ch6-cloud/ch6-07-crawler

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch6-cloud/ch6-07-crawler.html

**Comment by ZzzJing** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by changkaiyan** [comment]
您好，我将尽快回复您的邮件。如果您的邮件无需回复，则收到本回复表明我已收到您的邮件。请勿回复，谢谢合作。This is an automatic reply, confirming that your e-mail was received.Thank you.
